src/components/stepWizard/step-wizard.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import { Steps } from "./Steps";
+import { stepArray } from "./example usage/page";
+import Step from "./Step";
+
+export const stepWizard: React.FC = () => {
+  return (
+    <Steps totalSteps={3}>
+      {stepArray.map((step) => {
+        return <div>{step}</div>;

```

Comment:
There is an error in the console complaining about "each child in a list needing a unique 'key' prop". This can be fixed by giving the `div` unique `key` property. I'd say this could be approached two different ways:

1. If we think the list of steps won't change once created (we are not deleting/adding steps or changing step order), you can simply use the current step's index as the key.
2. If we think the list of steps could be modified, possibly we could give each step a unique ID? I see we are assigning stepNumber to the Step component, so maybe that can be used? We can also use unique strings for the key, if each step ends up with a unique name for example.